### ISSUE TITLE
Update Project Profile: remove Mario Antonini from HackforLA.org Website

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#7459]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
The Hackforla.org Website project page needs to be updated so that the project information is up to date. 

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) inside of the `_projects/website.md` file and find the `leadership` variable.
- [x] Delete the following lines from the file:
```
  - name: Mario Antonini
    github-handle: 
    role: Merge Team
    links:
      slack: 'https://hackforla.slack.com/team/U06MTPZ67V3'
      github: 'https://github.com/marioantonini'
    picture: https://avatars.githubusercontent.com/marioantonini
```
- [x] Verify the changes by viewing the [Hackforla.org Website project page](https://www.hackforla.org/projects/website) on your local environment and include a screenshot with your pull request

### Resources
- [website.md](https://github.com/hackforla/website/blob/gh-pages/_projects/website.md)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)

**Comment by HackforLABot** [comment]
Hi @codyjohnsontx, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by codyjohnsontx** [comment]
Working on it now. I have about an hour to complete this. Mainly working through the documentation so I don't mess it up.

**Comment by codyjohnsontx** [comment]
- **Status**: PR submitted
- **Documentation**: Worked through and completed
- **Time Taken**: Longest part was docs, working through code took about 5 minutes to find and delete lines
- **CodeQL**: No issues found, as far as I could tell
- **Screenshot**: Included to show changes
- **Next Steps**: Waiting on 2 reviews
